components/MyPage/ProfileSetting/index.tsx

```diff
@@ -4,16 +4,21 @@ import { ChangeEvent, useEffect, useState } from 'react'
 import { ApiType, OnDeleteType } from '@/type/components/MyPage'
 import useLoggedIn from '@/hooks/useLoggedIn'
 import { useRouter } from 'next/router'
+import TokenManager from '@/api/TokenManager'
 
 export default function ProfileSetting() {
   const [apiConfig, setApiConfig] = useState<ApiType>({
     url: '',
     method: 'get',
   })
+  const tokenManager = new TokenManager()
 
   const { fetch } = useFetch({
     url: apiConfig.url,
     method: apiConfig.method,
+    onSuccess: () => {
+      tokenManager.removeTokens()
+    },
```

Comment:
```suggestion
  const { fetch } = useFetch({
    url: apiConfig.url,
    method: apiConfig.method,
    onSuccess: () => {
      const tokenManager = new TokenManager()
      tokenManager.removeTokens()
    },
```

우린 next다 보니깐 초기 렌더링에서는 localStorage 값을 조회 할 수 없잖아
지금 내가 에러 처리를 해서 오류가 발생하지 않지만 사용 할거면 선언과 동시에 사용하는 걸로 해도 될 것 같아
저걸 굳이 전역으로 둘 필요는 없잖아

Reply:
ㅇㅋㅇㅋ 알겠어